scripts/templates/nxiq-operator-certified.clusterserviceversion.yaml

```diff
@@ -33,6 +33,12 @@ metadata:
             "iq": {
               "adminPort": 8071,
               "applicationPort": 8070,
+              "env": [
```

Comment:
Maybe should we also add the `INSTALL4J_ADD_VM_PARAMS` too? as we did here: https://github.com/Sonatype/operator-nxrm3/commit/1a4bed4f2e3d4ad02cbb9c2feb6358d0e5bd437a

Reply:
`INSTALL4J_ADD_VM_PARAMS` is the name nxrm used for `JAVA_OPTS`. this brings that same functionality to iq.

Reply:
oh I see, thanks for the clarification John 😄 Now it makes more sense to me